lib/src/browser.dart

```diff
@@ -57,20 +61,23 @@ class Browser {
     _firefox,
     _safari,
     _internetExplorer,
+    _edge,
     _wkWebView
   ];
 
   bool get isChrome => this == _chrome;
   bool get isFirefox => this == _firefox;
   bool get isSafari => this == _safari;
   bool get isInternetExplorer => this == _internetExplorer;
+  bool get isEdge => this == _edge;
```

Comment:
That's a good question. The only thing that I know for sure makes them different are the layout engines they run on. The layout engine Internet Explorer is Trident while the layout engine for Edge is EdgeHTML. I know as far as the canary tool we are building for support the distinction is important enough that it's a requirement for the project. So I know support has a strong opinion of the matter.